README.md

```diff
@@ -78,7 +78,7 @@ NODE_ENV=development
 SESSION_SECRET=anything
 PORT=3000
 SEND_LEGAL_MAIL_API_URL=http://localhost:8080
-PRISON_REGISTER_API_URL=http://localhost:8103
+PRISON_REGISTER_API_URL=https://prison-register-dev.hmpps.service.justice.gov.uk
```

Comment:
Just use the real service instead of an image. Not ideal but a) the real service is unsecured and b) the docker image needs yet another bunch of Localstack config when all we want is to serve up some prisons names and addresses.